<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PushAndReceiveWidget</class>
 <widget class="QWidget" name="PushAndReceiveWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_send">
   <property name="geometry">
    <rect>
     <x>385</x>
     <y>535</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>520</y>
     <width>370</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
background-color:white;
color: #333333; /* 字体颜色 */
padding: 5px; /* 内边距 */</string>
   </property>
   <property name="placeholderText">
    <string>来说点什么吧</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>521</height>
    </rect>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>369</x>
     <y>520</y>
     <width>81</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <zorder>textEdit</zorder>
  <zorder>frame</zorder>
  <zorder>pushButton_send</zorder>
  <zorder>listWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
